controllers/topics.controllers.js

```diff
@@ -1,9 +1,13 @@
-const { selectAllTopics } = require('../models/topics.models')
-
+const { selectApi, selectAllTopics } = require('../models/topics.models')
 
+exports.getApi = (req, res, next) => {
```

Comment:
You've got this in your topics controllers file, same as the model for this is in the topics models file. This could be confusing and hard to find later once this repo gets even busier.